.github/workflows/publish-package.yml

```diff
@@ -26,6 +30,15 @@ jobs:
     - name: Build a binary wheel and a source tarball
       run: |
         python -m pep517.build --source --binary --out-dir dist/ .
+    - name: Verify versioning is working
```

Comment:
shouldn't this only be run for non-tags? something like

`if: "!startsWith(github.ref, 'refs/tags')"`

Reply:
Actually, is this necessary to not have it run on some jobs? All it is doing is checking to make sure that the wheel name **does not** contain the string "pyhf-0.1.dev". If it doesn't then it doesn't matter.

Reply:
my point is that versioning should be confirmed by checking that the version is not "pyhf-0.1.dev" and that it matches "pyhf-*.dev" which is always true for non-tagged.